code/common/pubsub.q

```diff
@@ -73,13 +73,14 @@ selfiltered:{[x;y]
 
 // Publish table data
 pub:{[t;x]
-  if[not count x;:()];
-  if[count h:subrequestall[t];-25!(h;(`upd;t;x))];
-  if[t in subrequestfiltered`tbl;
-    {[t;x] data:eval(?;t;x`filts;0b;x`columns);neg[x`handle](`upd;t;data)}[t;] 
-      each select handle,filts,columns from subrequestfiltered where tbl=t
-  ];
- };
+    if[not count x;:()];
+    if[count h:subrequestall[t];-25!(h;(`upd;t;x))];
+    if[t in .stpps.subrequestfiltered`tbl;
+         {[t;x;sels] data:eval(?;x;sels`filts;0b;sels`columns);
+         //show count data;
+         if[count data;neg[sels`handle](`upd;t;data)]}[t;x;]
+    each select handle,filts,columns from .stpps.subrequestfiltered where tbl=t];
```

Comment:
the identation seems to be changed from before - previously the lambda was idented less & the `each ...` line was indented more, is there a reason to change this? It seems clearer to me the way it was before. (Also the placement of the closing `]` from the `if` and the closing `}` of the function)